src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-global-migration.xml

```diff
@@ -252,4 +252,17 @@
         </sql>
     </changeSet>
 
-</databaseChangeLog>
\ No newline at end of file
+    <!-- Global Scheduling Config -->
+    <changeSet author="psantos" id="1548361428926-5">
```

Comment:
Just to confirm: a config for this descriptor doesn't exist before this, right?

Reply:
Correct I tried to put the stored procedure call in the changelog-configuration-initialization.xml file first but no configuration exists for the settings component at the time that xml file is processed.  So I needed to do the same thing that was done with the other default configurations.